barman/postgres.py

```diff
@@ -556,7 +556,7 @@ def has_backup_privileges(self):
           OR
           (
             userepl
-            AND
+            OR
```

Comment:
For clarity, I think we should rewrite this to say `((userepl OR (…thing…)) AND …otherthings…)`, i.e., put the two OR clauses together inside (). Right now it's just `(userepl OR …thing… AND …otherthings…)`, and I don't think that's right.

Reply:
Added cosmetic changes to have a better global view